core/src/main/java/com/sequenceiq/cloudbreak/service/stack/StackDownscaleValidatorService.java

```diff
@@ -18,8 +19,13 @@
 
     public void checkInstanceIsTheClusterManagerServerOrNot(String instancePublicIp, InstanceMetadataType metadataType) {
         if (GATEWAY.equals(metadataType) || GATEWAY_PRIMARY.equals(metadataType)) {
-            throw new BadRequestException(String.format("Downscale for node [public IP: %s] is prohibited because it " +
-                            "serves as a host the Cluster Manager server", instancePublicIp));
+            String additionalIpPartForTemplate = "";
+            String messageTemplate = "Downscale for the given node%s is prohibited because it " +
```

Comment:
node %s

Reply:
if I make it that way that could cause a double whitespace between the words 'node' and 'is' if there is no public ip for the given node.